src/apps/ojph_compress/ojph_compress.cpp

```diff
@@ -461,9 +461,10 @@ int main(int argc, char * argv[]) {
 
     "The following option has a default value (optional):\n"
     " -num_decomps  (5) number of decompositions\n"
-    " -qstep        (1.0/255,0) quantization step size; all quantization\n"
-    "               step sizes are derived from this.\n"
-    " -reversible   (false) for irreversible adnd true for reversible\n"
+    " -qstep        (-1.0000...1.0000) quantization step size for lossy\n"
```

Comment:
-qstep  (0.0039 for 8-bit data) quantization step size; quantization step sizes for all subbands are derived from this value.  qstep values can be between a very small positive float number such as 0.00001 and 0.5.  Larger values produce smaller files (i.e., coarsely quantized coefficients).

I think there is no need to mention the -1.0; it is just an implementation detail that I might change.

Reply:
Too bad there is no printf. Could use ± sign

Reply:
I made changes as I understood.